IdentityService/PresentationLayer/Extentions/ServicesExtention.cs

```diff
@@ -0,0 +1,50 @@
+﻿using DataAccessLayer.Data;
+using DataAccessLayer.Models;
+using DataAccessLayer.Repositories;
+using Microsoft.EntityFrameworkCore;
+using BAL.Profiles;
+using BAL.Services;
+using Microsoft.AspNetCore.Server.IIS.Core;
+using System.Runtime.CompilerServices;
+
+namespace PresentationLayer.Extentions
+{
+    public static class ServicesExtention
+    {
+        public static IServiceCollection AddServices
+            (this IServiceCollection services, IConfiguration config)
+        {
+            services.AddScoped<IClientRepo, ClientRepo>();
+            services.AddScoped<ClientsService>();
+            services.AddAutoMapper(typeof(UserProfile).Assembly);
+            services.AddDbContext<UserDbContext>(options =>
+                options.UseInMemoryDatabase(databaseName: "InMemoryDatabase"));
+
+            //SeedData(services);
+            return services;
+        }
+
+        private static void SeedData(this IServiceCollection services)
+        {
+            using (var scope = services.BuildServiceProvider().CreateScope())
+            {
+                var dbContext = scope.ServiceProvider.GetRequiredService<UserDbContext>();
+
+                if (!dbContext.Clients.Any())
+                {
+                    Console.WriteLine("--> Seeding Data...");
```

Comment:
???????????

Reply:
at this stage of development I use InMemory db and seeding data to check my endpoints